### PR TITLE
WIP: Use dep instead of glide for vendors management

### PR DESCRIPTION
**Pull request by jaybatra26**
Signed off by Jay Batra <jaybatra73@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes # https://github.com/kubernetes/cloud-provider-azure/issues/101

**Special notes for your reviewer**:

**Release note**:
```
This change will let users use go dep instead of glide from now on
```

### CODE REVIEW

File: _vendor-20190305212256/github.com/Azure/azure-sdk-for-go/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (feiskyer) [inline _vendor-20190305212256/github.com/Azure/azure-sdk-for-go/LICENSE]: this dir should be removed

**Comment by feiskyer** [comment]
After the updates, the versions are wrong, hence unit checks failed.

@jaybatra26 Could you recheck the vendor updates?

**Comment by jaybatra26** [comment]
@feiskyer Looking into it, will possibly create a new PR

**Pull request closed without merging**